core/foundation/inc/ROOT/RStringView.hxx

```diff
@@ -55,7 +55,7 @@ namespace std {
 
 #endif // ifdef else R__HAS_STD_STRING_VIEW
 
-#ifndef R__HAS_OP_EQUAL_PLUS_STRING_VIEW
+#if !defined(R__HAS_OP_EQUAL_PLUS_STRING_VIEW) && !( defined(_MSVC_LANG) && _MSVC_LANG >= 201703L ) && ! ( __cplusplus >= 201703L )
```

Comment:
If you don't have an opinion then I think I prefer what I suggested here :-) It's more compact and keeps the symmetry with the other change, so I even have arguments.

Reply:
Sorry, was very busy with family and missed that this one was not closed.